### PR TITLE
add: animation prop 추가함

### PR DESCRIPTION
**Pull request by bclabs-kate**
# Issue
link url https://bclabs.atlassian.net/browse/PLAYV-1059

# What fix
- animation props 추가

[PLAYV-1059]: https://bclabs.atlassian.net/browse/PLAYV-1059?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bclabs-paul — APPROVED** [review]
🐏🐏🐏🐏🐏

**Review by bclabs-theo — APPROVED** [review]
💯💯💯💯💯

File: src/components/Alert/Alert.tsx [hunk 1]
```diff
@@ -32,35 +33,47 @@ const Alert: React.FC<AlertProps> = ({
     default:
       throw Error('invalid color value');
   }
+  console.log(color, animation);
```
Comment (bclabs-mia) [inline src/components/Alert/Alert.tsx:36]: 불필요한 로그는 지워주세요~!

**Review by bclabs-paul — APPROVED** [review]
🪶🪶🪶🪶🪶

**Review by bclabs-mia — APPROVED** [review]

**Pull request merged**